docs/quickstart-aks.md

```diff
@@ -144,7 +112,7 @@ Query the name servers of the zone with the following command and configure the
 
 Please note, it can take a while for the DNS changes to propogate.
```

Comment:
typo: propagate

Is there a way for the user to check if the changes have propagated? If yes, maybe mention that here.

Reply:
I believe there isn't an easy way for us to determine if he DNS changes have propagated across all the systems. On the developers system the user could do a ping or dig, etc. But that would not mean that the changes have been propagated for the lets encrypt server to validate the certificate request. While we're on the topic I did come across https://dnschecker.org/, do you think it makes sense to advise the user to use it?

Reply:
Yes, it might make sense to add that. We have something similar in our docs, see https://docs.bitnami.com/general/apps/wordpress/administration/update-ip-hostname/ for an example.

Reply:
The issue is LetsEncrypt will not issue us any certificates until the DNS propagation is completed. The `/etc/hosts` method resolves the DNS issue locally, but LetsEncrypt is an external component which will wait for the DNS propagation to actually complete.

I spoke to other members of the team re: dnschecker.org and we came to a conclusion that it does not help the case because it really does not fully indicate that the propagation has completed (just because the changes are reflected on their servers, doesn't mean that they have propagated every where.)

Reply:
OK